res/values-v14/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+
+    <dimen name="widget_margin">0dp</dimen>
```

Comment:
Why does this need a different padding on API14+? Do you have a link to the documentation for this?
Also, this is used as padding, not margin, so the name should reflect that.